ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockClusterMap.java

```diff
@@ -35,6 +35,7 @@
   private final List<MockDataNodeId> dataNodes;
   private final int numMountPointsPerNode;
   private final HashSet<String> dataCentersInClusterMap = new HashSet<>();
+  private boolean partitionsUnhealthy = false;
```

Comment:
what is this used for ? Why do we need this. In real system, do we have a scenario for a partition being unhealthy ?